lib/param_parsing/parameters.go

```diff
@@ -45,6 +46,7 @@ func GetParameters() Params {
 	getopt.BoolVarLong(&params.ShowLatestFlag, "show-latest", 'U', "Show latest stable version")
 	getopt.StringVarLong(&params.ShowLatestPre, "show-latest-pre", 'P', "Show latest pre-release implicit version. Ex: tfswitch --show-latest-pre 0.13 prints 0.13.0-rc1 (latest)")
 	getopt.StringVarLong(&params.ShowLatestStable, "show-latest-stable", 'S', "Show latest implicit version. Ex: tfswitch --show-latest-stable 0.13 prints 0.13.7 (latest)")
+	getopt.BoolVarLong(&params.ShowSelectedFlag, "show-selected", 'w', "Display the currently selected terraform version by tfswitch")
```

Comment:
Where does the `w` option come from? 

Personally I would combine this with the `-v` option so it shows both the tfswitch version, and the currently selected Terraform version. I think this pattern is fairly common, where `-v` supplies versions for related components.